src/rust-crypto/backup.ts

```diff
@@ -26,9 +26,9 @@ import { encodeUri } from "../utils";
 import { OutgoingRequestProcessor } from "./OutgoingRequestProcessor";
 import { sleep } from "../utils";
 
-/* eslint-disable camelcase */
 interface PreparedKeyBackupVersion {
     algorithm: string;
+    /* eslint-disable camelcase */
```

Comment:
I think this will disable the lint rule for the rest of the file. It needs `eslint-disable-next-line`

Reply:
done